### PR TITLE
Typing adjustment - 1357

### LINKED ISSUE #1357: Webapp Linter Fixes - fix miscellaneous  no-explicit-any warnings
The following files each contain a no-explicit-any warnings.
	* Run `npx eslint --ext .tsx,.ts .` in the webapp folder to see them

webapp/src/telemetry/telemetryClient.ts
webapp/src/testUtils.tsx
webapp/src/blocks/archive.ts

### PR DESCRIPTION
**Pull request by tiago154**
#### Summary

Hello.

Adjusted the warnings in the files
`webapp/src/telemetry/telemetryClient.ts`
`webapp/src/testUtils.tsx`
`webapp/src/blocks/archive.ts`

where the typing from `any` to `unknown` was changed.

#### Ticket Link

Fixes #1357

### CODE REVIEW

File: webapp/src/testUtils.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function mockDOM(): void {
     }
 }
 
-export function mockMatchMedia(result: any): void {
+export function mockMatchMedia(result: unknown): void {
```
Comment (hahmadia) [inline webapp/src/testUtils.tsx:27]: Hello!

I believe this type can be more specific

```suggestion
export function mockMatchMedia(result: {matches: boolean}): void {
```

If we look at were the function is being called, this is the only thing being passed into it.

**Review by hahmadia — CHANGES_REQUESTED** [review]
Hey! Thanks for the PR. 

Small feedback.

File: webapp/src/testUtils.tsx (hunk 1 above)
Comment (tiago154) [reply webapp/src/testUtils.tsx:27]: You are right. I applied the suggestion.

**Review by hahmadia — APPROVED** [review]

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @tiago154

**Pull request merged**